src/main/java/com/softserveinc/dokazovi/entity/UserEntity.java

```diff
@@ -70,6 +72,20 @@ public class UserEntity {
 	@CreationTimestamp
 	private Timestamp createdAt;
 
+	@EqualsAndHashCode.Exclude
```

Comment:
Please advise what was the reason of applying `@EqualsAndHashCode.Exclude` annotation here?